crypto/envelope_detector.go

```diff
@@ -196,6 +196,9 @@ func (wrapper *OldContainerDetectorWrapper) OnCryptoEnvelope(ctx context.Context
 
 // OnColumn callback which finds serializedContainer or AcraStruct/AcraBlock for backward compatibility
 func (wrapper *OldContainerDetectorWrapper) OnColumn(ctx context.Context, inBuffer []byte) (context.Context, []byte, error) {
+	if inBuffer == nil {
```

Comment:
@Lagovas Can we place this check on the level of `EnvelopeDetector`? We are already doing some `inBuffer` checking [here](https://github.com/cossacklabs/acra/blob/master/crypto/envelope_detector.go#L60)

Reply:
but we return there `inBuffer` as is. Without allocating new buffer. If `inBuffer` in OnColumn will be `nil`, it will returned as is due to `if len(inBuffer) < SerializedContainerMinSize`

Reply:
Got it;